RawVideo/VideoStreamRenderer.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Azure.Communication.Calling.WindowsClient;
+using System;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace RawVideo
+{
+    internal class VideoStreamRenderer
```

Comment:
Why this class needs to be inline?

Reply:
sorry what do you mean with inline class?